Dockerfile

```diff
@@ -9,19 +9,15 @@ ADD https://raw.githubusercontent.com/vishnubob/wait-for-it/master/wait-for-it.s
 RUN chmod +x /usr/local/bin/wait-for-it
 
 # Copy package.json and package-lock.json to the working directory.
-COPY package.json package-lock.json ./
-
-# Copy all project files to the working directory, excluding those in .dockerignore.
-COPY . .
-
-# Copy the Docker-specific environment file.
-COPY .env.docker .env
+COPY package*.json ./
 
 # Install app dependencies.
-RUN npm cache clean --force && rm -rf node_modules && npm install && npm rebuild bcrypt 
+RUN npm install

```

Comment:
the reason I have used the command is that the `bcrypt` package was causing issues. Were you able to solve it?